exporter/kafkaexporter/kafka_exporter.modified.go

```diff
@@ -135,7 +135,6 @@ func newSaramaProducer(config Config) (sarama.SyncProducer, error) {
 	c.Metadata.Full = config.Metadata.Full
 	c.Metadata.Retry.Max = config.Metadata.Retry.Max
 	c.Metadata.Retry.Backoff = config.Metadata.Retry.Backoff
-	configureCompression(config.Compression, c)
```

Comment:
@laxman-traceable @pavan-traceable Can you verify this is ok?

Reply:
@tim-mwangi  I am not the right person, @ravisingal can you please check this ?

Reply:
cc @laxmanchekka

Reply:
Looks good to me.

cc: @jcchavezs @tim-mwangi